src/sass/index.sass

```diff
@@ -123,20 +126,22 @@
   #pause
     display: none
   
+  #menu
+    top: auto !important
+
   .window-app
     display: flex !important
     flex-direction: column !important
     width: 100% !important
     min-width: 0 !important
     min-height: 0 !important
-    top: 0 !important
+    top: auto !important
```

Comment:
Not all windows should be glued to the top, so I only added `top: 0` for actor sheets